pkg/nodeinfo/attributes.go

```diff
@@ -36,17 +36,22 @@ const (
 	NodeLabelMlnxNIC       = "feature.node.kubernetes.io/pci-15b3.present"
 	NodeLabelNvGPU         = "nvidia.com/gpu.present"
 	NodeLabelWaitOFED      = "network.nvidia.com/operator.mofed.wait"
+	NodeLabelCudaVersion   = "nvidia.com/cuda.driver.major"
 )
 
 type AttributeType int
```

Comment:
An alternative would be to extend this to be a struct where we are able to mark an attribute as optional. current approach is less intrusive though.

Reply:
Yes, I was also thinking about a similar approach which using stuct to hold information about is the option is required or not. The approach with struct is much wordy and for me, it decreases readability a bit. I prefer to stick with the current implementation.